CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
+if( NOT SKIP_GOOGLETEST_INSTALL )
+   set( ENABLE_GOOGLETEST_INSTALL ON )
```

Comment:
@d3x0r http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html Please squash everything into one commit (and clean all the stray spaces as @mwoehlke-kitware hinted at)